src/H5FDmulti.h

```diff
@@ -25,11 +25,228 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-H5_DLL hid_t  H5FD_multi_init(void);
+H5_DLL hid_t H5FD_multi_init(void);
+
+/**
+ * \ingroup FAPL
+ *
+ * \brief Sets up use of the multi-file driver
+ *
+ * \fapl_id
+ * \param[in] memb_map Maps memory usage types to other memory usage types
+ * \param[in] memb_fapl Property list for each memory usage type
+ * \param[in] memb_name Name generator for names of member files
+ * \param[in] memb_addr The offsets within the virtual address space, from 0
+ *           (zero) to #HADDR_MAX, at which each type of data storage begins
+ * \param[in] relax Allows read-only access to incomplete file sets when \c TRUE
+ * \returns \herr_t
+ *
+ * \details H5Pset_fapl_multi() sets the file access property list \p fapl_id to
+ *          use the multi-file driver.
+ *
+ *          The multi-file driver enables different types of HDF5 data and
+ *          metadata to be written to separate files. These files are viewed by
+ *          the HDF5 library and the application as a single virtual HDF5 file
+ *          with a single HDF5 file address space. The types of data that can be
+ *          broken out into separate files include raw data, the superblock,
+ *          B-tree data, global heap data, local heap data, and object
+ *          headers. At the programmer's discretion, two or more types of data
+ *          can be written to the same file while other types of data are
+ *          written to separate files.
+ *
+ *          The array \p memb_map maps memory usage types to other memory usage
+ *          types and is the mechanism that allows the caller to specify how
+ *          many files are created. The array contains #H5FD_MEM_NTYPES entries,
+ *          which are either the value #H5FD_MEM_DEFAULT or a memory usage
+ *          type. The number of unique values determines the number of files
+ *          that are opened.
+ *
+ *          The array \p memb_fapl contains a property list for each memory
+ *          usage type that will be associated with a file.
+ *
+ *          The array \p memb_name should be a name generator (a
+ *          \Code{printf}-style format with a \Code{%s} which will be replaced
+ *          with the name passed to H5FDopen(), usually from H5Fcreate() or
+ *          H5Fopen()).
+ *
+ *          The array \p memb_addr specifies the offsets within the virtual
+ *          address space, from 0 (zero) to #HADDR_MAX, at which each type of
+ *          data storage begins.
+ *
+ *          If \p relax is set to TRUE (or 1), then opening an existing file for
```

Comment:
Should "TRUE (or 1)" or "1 (TRUE)" be used consistently throughout?  I think we should follow the existing style.